src/status_im2/contexts/wallet/events.cljs

```diff
@@ -75,3 +77,71 @@
                            data-store/<-rpc)
                      data)}]
      {:db (assoc db :wallet/networks network-data)})))
+
+(def collectibles-request-batch-size 1000)
+
+(defn- displayable-collectible?
```

Comment:
can we include some tests for these events and subs added in this pr? At least to cover the main functionality 👌